### PR TITLE
Remove unused strings, add one accessibility string

### PR DESCRIPTION
**Pull request by liuche**
This is just to push some string deletions, and one new. There's no deadline on the new string, we'll just merge it in as it gets translated.

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -27,6 +27,10 @@ msgctxt "firefox_tv_brand_name"
 msgid "Firefox TV"
 msgstr ""
 
+msgctxt "firefox_tv_brand_name_short"
+msgid "Firefox"
```
Comment (Delphine) [inline locales/templates/app.pot:31]: In the other string you add here, %1$s  is used for the brand name.
Shouldn't it be the case here as well?

**Review by Delphine — COMMENTED** [review]
I think other than that we should be good to go :) thanks

File: locales/templates/app.pot (hunk 1 above)
Comment (liuche) [reply locales/templates/app.pot:31]: I think in this case, "Firefox" is the short name, and "Firefox TV" is the full name - I'd consider them separate strings, depending on which we want to use.

Comment (Delphine) [reply locales/templates/app.pot:31]: oh ok, the other string's comment reads "Content description for the Exit Firefox button.   %1$s will be replaced
with the Firefox brand name." so I assumed they were the same.
I can go ahead and merge then, thanks

**Pull request merged**